pkg/kf/commands/install/kf/kf.go

```diff
@@ -49,28 +50,26 @@ func Install(ctx context.Context, containerRegistry string) error {
 		{name: "Knative Build", yaml: KnativeBuildYAML},
 		{name: "kf", yaml: KfNightlyBuildYAML},
 	} {
-
-		var err error
-		for i := 0; i < 10; i++ {
+		wait.ExponentialBackoff(wait.Backoff{
```

Comment:
I think this needs to quit if it never succeeded.

Reply:
Not quite:

> It checks the condition up to Steps times
[docs](https://godoc.org/k8s.io/apimachinery/pkg/util/wait#ExponentialBackoff)

Some "physical" proof:
https://play.golang.org/p/gXKYR9LnFVO

Reply:
I think @josephlewis42  is saying we should check the error returned by wait.ExponentialBackoff
If it's not nil then we should fail fast and not continue to apply yaml files

Reply:
https://play.golang.org/p/cU2JNRxi_8W